src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFHeaderLines.java

```diff
@@ -113,7 +113,7 @@ public static VCFFormatHeaderLine getEquivalentFormatHeaderLine(final String inf
         addFormatLine(new VCFFormatHeaderLine(MLE_PER_SAMPLE_ALLELE_FRACTION_KEY, VCFHeaderLineCount.A, VCFHeaderLineType.Float, "Maximum likelihood expectation (MLE) for the alternate allele fraction, in the same order as listed, for each individual sample"));
         addFormatLine(new VCFFormatHeaderLine(PL_FOR_ALL_SNP_ALLELES_KEY, 10, VCFHeaderLineType.Integer, "Phred-scaled genotype likelihoods for all 4 possible bases regardless of whether there is statistical evidence for them. Ordering is always PL for AA AC CC GA GC GG TA TC TG TT."));
         addFormatLine(new VCFFormatHeaderLine(HAPLOTYPE_CALLER_PHASING_ID_KEY, 1, VCFHeaderLineType.String, "Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group"));
-        addFormatLine(new VCFFormatHeaderLine(HAPLOTYPE_CALLER_PHASING_GT_KEY, 1, VCFHeaderLineType.String, "Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another"));
+        addFormatLine(new VCFFormatHeaderLine(HAPLOTYPE_CALLER_PHASING_GT_KEY, 1, VCFHeaderLineType.String, "Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another; will always be heterozgyous and is not intended to describe called alleles"));
```

Comment:
`heterozgyous` -> `heterozygous`

Reply:
I can't even get a one-line PR right.

Reply:
It's my fault @ldgauthier for claiming that it would be a one-line PR. I forgot that the `HaplotypeCaller` exact-match tests would fail due to the header differences (https://storage.googleapis.com/hellbender-test-logs/build_reports/master_27820.4/tests/test/index.html). The good news is that these days we have that `UPDATE_EXACT_MATCH_EXPECTED_OUTPUTS` boolean you can turn on in `HaplotypeCallerIntegrationTest` to auto-update the offending files.